man/el.man

```diff
@@ -0,0 +1,60 @@
+.TH VRMS-RPM 1
+.SH NAME
+vrms-rpm - αναφορά εγκατεστημένου μη-ελεύθερου λογισμικού
+.SH ΣΥΝΟΨΗ 
+\ fBvrms-στροφές \ fR \ fI [ΕΠΙΛΟΓΗ] \ fR ...
+.SH ΠΕΡΙΓΡΑΦΗ
+Αυτό το πρόγραμμα είναι ένας κλώνος του πρωτοτύπου
+\ fBvrms \ fR ("\ fBvirtual Richard M. Stallman \ fR")
+για το Debian GNU / Linux, το οποίο δημιουργήθηκε με σκοπό να φέρει το δικό του
+λαμπρή ιδέα για τις διανομές GNU / Linux που βασίζονται σε RPM.
+.PP
+Αυτό το πρόγραμμα αναλύει τη λίστα των πακέτων RPM και των αναφορών που είναι εγκατεστημένα
+βρήκε μη-ελεύθερα πακέτα σε stdout. Η απόφαση σχετικά με το εάν θα ταξινομηθεί ένα πακέτο
+ως δωρεάν ή μη, βασίζεται στις οδηγίες αδειοδότησης του έργου Fedora.
+.PP
+Οι χρήστες του Fedora / CentOS που χρησιμοποιούν επιπλέον αποθετήρια, όπως το \ fBRPM Fusion \ fR,
+μπορεί να παρατηρήσει ότι δεν είναι όλα τα πακέτα που προέρχονται από μη επίσημα repos σημειωμένα ως
+μη-δωρεάν. Αυτό οφείλεται στο γεγονός ότι η διάκριση, όπως σημειώθηκε προηγουμένως, γίνεται ανάλογα
+στην άδεια συσκευασίας. Υπάρχουν διάφοροι λόγοι, πέραν της χορήγησης αδειών, γιατί
+ένα πακέτο μπορεί να μην έχει γίνει αποδεκτό στον υπάλληλο
+Fedora / CentOS repository - όπως οι ευρεσιτεχνίες λογισμικού ή οι περιορισμοί εξαγωγής.
+.PP
+Ο Richard Stallman ήταν, εδώ και χρόνια, η πιο εξέχουσα και φωνητική φιγούρα
+στο κίνημα Ελεύθερου Λογισμικού, υπενθυμίζοντας αδιάκοπα σε θέματα του
+ηθική λογισμικού. Όπως και ο Richard, πιστεύουμε ότι οι χρήστες πρέπει να έχουν έναν εύκολο τρόπο
+γνωρίζουν αν εκτελούν μη-ελεύθερο λογισμικό στα συστήματά τους.
+Αυτό το πρόγραμμα είναι μια προσπάθεια να ικανοποιηθεί αυτή η ανάγκη.
+.SH ΕΠΙΛΟΓΕΣ
+.TP
+\ fB \ - \ - ascii \ fR
```

Comment:
`\fB` and `\fR` need to be fixed. The same way for `\-` - no space after the slash.